src/main/java/com/obj/nc/controllers/DeliveryInfoRestController.java

```diff
@@ -70,8 +71,11 @@ public class DeliveryInfoRestController {
 	public ResultPage<EndpointDeliveryInfoDto> findDeliveryInfosByEventId(
 			@PathVariable (value = "eventId", required = true) String eventId,
 			@RequestParam(value = "endpointId", required = false) String endpointId,
-			@PageableDefault(size = 20) Pageable pageable
+			@RequestParam("page") int page,
```

Comment:
Preco bolo treba zmenit signaturu? koly tomu ze v anotacii nevies nastavit default? .. neni ta to nejaka springova buildin property?

Reply:
ked tu mame Pageable pageable, tak nedokazem ocheckovat boundaries (a nasledne vyhlasit chybu) preto, ze spring mi ich zmeni este predtym, nez vojdu do tejto metody cez nejake automaticke pageable a sort resolvery (jedine ze by som overridol 4-5 springovych beanov ktore suviasia s resolvingom, ale to nechcem robit)

cielom nie je nastavit default, ale mat moznost checknut ci to co Martin zada do localhost:8080/nc/get-delivery-infos?page=0&size=300 tak mu odpovedat ze sa pyta zle, ked je indexovanie od 1 a ked spring ma nastavenu max velkost stranky 200, aby sa necudoval, preco mu to orezava na 200
springove properties na default su (tym zabalenim do web som sa inspriroval prave tu):
spring.data.web.pageable.one-indexed-parameters=true
spring.data.web.pageable.max-page-size=2000
ale radsej som si vytvoril explicitne vlastne, ktore pouzivaju len moje objekty